keras_cv/core/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_cv.core.factor_sampler.constant_factor_sampler import ConstantFactorSampler
```

Comment:
API wise, do we want to have 'keras_cv.ConstantFactorSampler' "keras_cv.core.ConstantFactorSampler" and "keras_cv.core.factor_sampler.ConstantFactorSampler" at the same time?

Reply:
I guess we can remove the core __init__ exports because users should never use that namespace.

Reply:
Updated to just be keras_cv.*